src/SFA.DAS.EmployerIncentives.Domain/ValueObjects/Incentive.cs

```diff
@@ -169,6 +169,8 @@ public class Phase2Incentive : Incentive
         {
         }
 
+        public static int MinimumPhase2AgreementVersion => 6;
```

Comment:
why the "Phase2" when it's implied?

Reply:
You can't have a method and a property with the same name, so had to rename it. Changed to use the overloaded method.